packages/serialize-request/test/unit/lib/index.spec.js

```diff
@@ -220,6 +220,19 @@ describe('@dotcom-reliability-kit/serialize-request', () => {
 				});
 			});
 		});
+		describe('when `request.headers` is iterable', () => {
+			beforeEach(() => {
+				request.headers = new Map([
+					['Content-Type', 'application/json'],
+					['Authorization', 'Bearer token']
+				]);
+			});
+			it('should handle headers as an iterable', () => {
+				expect(serializeRequest(request)).toMatchObject({
+					headers: { 'content-type': 'application/json' }
+				});
+			});
+		});
```

Comment:
**suggestion:** this test is great 🙂 I wonder whether we should also be certain about the string checking to make sure this can't regress? Maybe another test like:

```suggestion
		});

		describe('when `request.headers` is iterable but has non-string values', () => {
			beforeEach(() => {
				request.headers = new Map([
					['Content-Type', 123],
					['accept', {}]
				]);
			});
			it('should ignore the headers with non-string values', () => {
				expect(serializeRequest(request)).toMatchObject({
					headers: {}
				});
			});
		});
```